src/client/method-status.c

```diff
@@ -283,3 +283,7 @@ int method_status_unit_on(Client *client, char *node_name, char **units, size_t
 
         return 0;
 }
+
+int method_status(Command *command, void *userdata) {
+        return method_status_unit_on(userdata, command->opargv[0], &command->opargv[1], command->opargc - 1);
```

Comment:
Should `method_status_unit_on` be `static` now?

Reply:
Yes, thanks